fabric-client-tags-api-v1/src/testmodClient/java/net/fabricmc/fabric/test/tag/client/v1/ClientTagTest.java

```diff
@@ -20,20 +20,33 @@
 import org.slf4j.LoggerFactory;
 
 import net.minecraft.block.Blocks;
+import net.minecraft.registry.Registries;
+import net.minecraft.registry.tag.TagKey;
+import net.minecraft.util.Identifier;
 import net.minecraft.world.biome.BiomeKeys;
 
 import net.fabricmc.api.ClientModInitializer;
 import net.fabricmc.fabric.api.client.event.lifecycle.v1.ClientLifecycleEvents;
+import net.fabricmc.fabric.api.client.event.lifecycle.v1.ClientTickEvents;
+import net.fabricmc.fabric.api.resource.ResourceManagerHelper;
+import net.fabricmc.fabric.api.resource.ResourcePackActivationType;
 import net.fabricmc.fabric.api.tag.client.v1.ClientTags;
 import net.fabricmc.fabric.api.tag.convention.v1.ConventionalBiomeTags;
 import net.fabricmc.fabric.api.tag.convention.v1.ConventionalBlockTags;
 import net.fabricmc.fabric.api.tag.convention.v1.ConventionalEnchantmentTags;
+import net.fabricmc.loader.api.FabricLoader;
 
 public class ClientTagTest implements ClientModInitializer {
 	private static final Logger LOGGER = LoggerFactory.getLogger(ClientTagTest.class);
+	private static final String MODID = "fabric-clients-tags-api-v1-testmod";
 
 	@Override
 	public void onInitializeClient() {
+		FabricLoader.getInstance().getModContainer(MODID)
+				.map(container -> ResourceManagerHelper.registerBuiltinResourcePack(new Identifier(MODID, "test2"),
+						container, ResourcePackActivationType.ALWAYS_ENABLED))
+				.filter(success -> !success).ifPresent(success -> LOGGER.warn("Could not register built-in resource pack."));
```

Comment:
```suggestion
		final ModContainer container = FabricLoader.getInstance().getModContainer(MODID).get();

		if (!ResourceManagerHelper.registerBuiltinResourcePack(new Identifier(MODID, "test2"),
				container, ResourcePackActivationType.ALWAYS_ENABLED)) {
			throw new IllegalStateException("Could not register built-in resource pack.");
		}
```

Id perfer to fail hard if something hasnt gone quite right, especially in a test.